layouts/shortcodes/release-binaries.html

```diff
@@ -0,0 +1,93 @@
+<!-- Fetch release_binaries.json from kubernetes-sigs/downloadkubernetes to render in table -->
+{{ $response := getJSON "https://raw.githubusercontent.com/dipesh-rawat/downloadkubernetes/integrate-k8s-website/dist/release_binaries.json" }}
```

Comment:
I'm wondering for how long this JSON file will be maintained.

Reply:
The JSON file will be updated automatically when the sig-release team applies the usual refresh in repo [kubernetes-sigs/downloadkubernetes](https://github.com/kubernetes-sigs/downloadkubernetes) upon every new versions (patch/minor). The shortcode will pick up the JSON file and render the latest data accordingly.

Reply:
@dipesh-rawat: how do we know that the release team are maintaining your fork of downloadkubernetes?

Perhaps we can use a more official artifact?

Reply:
If you want cross-site scripting preflighting set up for downloadkubernetes.com, we probably can make that happen. Or we can fetch the data at site build time.

Reply:
I used JSON file from my fork to generate the deploy preview while waiting for the release team, and now that the PR [kubernetes-sigs/downloadkubernetes#472](https://github.com/kubernetes-sigs/downloadkubernetes/pull/472/files) has been approved and merged by the release team, I've updated the shortcode to use the file from the official Kubernetes [kubernetes-sigs/downloadkubernetes](https://github.com/kubernetes-sigs/downloadkubernetes) repo.